src/assertions/OrderAssertions.ts

```diff
@@ -195,7 +268,7 @@ export class OrderAssertions {
     );
 
     // The 0x taker token is equivalent to the issuance order maker token
-    this.commonAssertions.isEqualString(
+    this.commonAssertions.isEqual(
```

Comment:
Create a new common assertion called `isEqualAddress` that asserts string equivalance